### ISSUE TITLE
Not being notified when a layer/doc/map is seen

### ISSUE THREAD

**Issue by meyerch** [geosolutions-it/unesco-ihp#125]
I noticed that in the "Notifications" list (both here http://ihp-wins-dev.geo-solutions.it/notifications/settings/ and here http://ihp-wins-dev.geo-solutions.it/admin/pinax_notifications/noticetype/), there is no "Layer/doc/map seen" notification. 
However, users receive an email whenever one of their layer/doc/map is seen. 
So the issue is that this notifcation is activated by default, but cannot be deactivated by the user. 

--> in the notifications list, can you add:
- Layer seen [A layer was seen]
- Document seen [A document was seen]
- Map seen [A map was seen]
so that users can choose whether or not they want to be notified for this.

**Comment by simboss** [comment]
Hi @meyerch if I understood correctly you would want to have users notified when someone opens one of their resources, correct?

If my understanding is correct, I believe this should be part of a separate contract as it is not a minor fix and goes a little beyond the scope of the initial plan.

**Comment by meyerch** [comment]
Hi @simboss 
No, it is the opposite acctually. 
At the moment, users are notified whenever someone "sees" their layer/map/document (see below an example of an email I receive whenever someone click on one of my layers/maps/documents)
 
However, I realised that this type of notification is not in the notification list: users cannot choose not to be notified by email for this. 
So basically, the issue is that users are forced to receive email notifications whenever someone sees their resource. 

So my question was: can you allow users to choose whether or not they want to be notified for this?
In other words: can you add "Layer/doc/map seen" in the notifications list, so that users can chose to tick it or untick it? 

Let me know if it is still not clear

**Comment by simboss** [comment]
Interesting, I have asked @afabiani to dig further.

**Comment by meyerch** [comment]
@simboss  and @afabiani 
I forgot to specify that this is something that happens for sure on the public instance. 

I don't receive notifications from the dev instance when my layers are seen, but I guess this is because the email notifications from your server are filtered out (?).

**Comment by afabiani** [comment]
@meyerch  @simboss 
so currently I could not reproduce the issue on DEV but I have found an explicit method on GeoNode core code (outside the Notifications mechanism) looking for owner and viewer emails and trying to send a message accordingly.

For the moment I commented out that code, since, as @meyerch suggested, it should be handled correctly through the Notifications. This would require some work, so for now I just keep it disabled.

In order to test this on production, we can:
1. Wait until we port DEV changes on PROD
2. I can port on PROD just this fix in order to test it before the deployment

Please, let me know how you prefer to proceed.

**Comment by meyerch** [comment]
hi @afabiani & @simboss 
Thanks for the message. 
I see no problem in keeping it disabled for the moment, so I prefere that we wait until we port DEV changes on PROD to fix this. 

So there is only task #124 before deploying the changes.
